sdk/core/azure-servicemanagement-legacy/tests/legacy_mgmt_testcase.py

```diff
@@ -82,13 +82,18 @@ def create_service_management(self, service_class):
         return service
 
     def _create_storage_service(self, service_class, settings, account_name=None, account_key=None):
-        account_name = account_name or settings.STORAGE_ACCOUNT_NAME
-        account_key = account_key or settings.STORAGE_ACCOUNT_KEY
+        try:
+            account_name_arg = account_name or settings.STORAGE_ACCOUNT_NAME
+        except NameError:
+            account_name_arg = settings.STORAGE_ACCOUNT_NAME
+        try:
+            account_key_arg = account_key or settings.STORAGE_ACCOUNT_KEY
+        except NameError:
+            account_key_arg = settings.STORAGE_ACCOUNT_KEY
         session = Session()
         service = service_class(
-            account_name,
-            account_key,
-            request_session=session,
+            "https://{}.blob.core.windows.net".format(account_name_arg),
+            credentials=account_key_arg
         )
         self._set_service_options(service, settings)
```

Comment:
make it static?

Reply:
This is not part of storage SDK - so we can ignore inefficiencies here for the moment